data/campaigns/Descent_Into_Darkness/scenarios/01_Saving_Parthyn.cfg

```diff
@@ -16,30 +16,37 @@
+            story= _ "By the time Malin returned to Parthyn, orcs had already begun to attack the city. Although he tried his best to fight them off, the raids were far more relentless and the town guard was far weaker than he had remembered. One night, a particularly powerful strike stormed the city and Malin could only watch as his father was cut down by one of the savage orcish warriors."
             {STORYTXT_BACKGROUND_BIGMAP_DID}
         [/part]
         [part]
-            # wmllint: local spelling misdoubt
-            story= _ "Shortly a man dressed in the robes of a necromancer emerged from the trees. <i>“Allow me to introduce myself. I am Darken Volk. While I know your kingdom does not love my kind, I think it vital that we put aside our differences for the moment. The orc menace grows in the North, and I misdoubt any of the border towns can stand without some assistance. For now, I ask only to rest in your town for a few days.”</i>"
+            story= _ "Shaken by the loss of their commander, the human defense began to falter. The orcs roared in anticipation of victory, but before they could overrun the city, skeleton warriors poured out of the woods to their rear. The undead cut down orc after orc, swords and axes hacking through flesh and bone with ruthless efficiency. The soldiers watched on in half amazement, half fear as the orcs dropped and the skeletons turned toward them, empty eyes bleeding with fetid magic. All was dreadfully still as the blood and carnage settled in the rank darkness."
             {STORYTXT_BACKGROUND_BIGMAP_DID}
         [/part]
         [part]
-            story= _ "Drogan, the captain of the guard, replied, <i>“Surely you know that the penalty for necromancy is death. For your aid we will promise you rest and safety. Upon your departure, however, you are banished from these lands, not to return on pain of death. We thank you for your help, but need no alliances with those who deal only in dark magic.”</i>"
+            story= _ "The silence was broken by the irregular shuffle of a man emerging from the trees. He stood before them, a hunched figure with a rasping voice reminiscent of crackling bones. <i>“I am Darken Volk. While I know you do not love my kind, I think it is vital that we put aside our differences for the moment. The orc menace grows in the north, and they will soon return in even greater force. You cannot hope to defend against them without my help.”</i>"
+            {STORYTXT_BACKGROUND_BIGMAP_DID}
+        [/part]
+        [part]
+            story= _ "Drogan, the captain of the guard, replied, <i>“You surely know that in these lands, the penalty for necromancy is death. As we do not currently possess the means to fight you, and seeing that you have saved our village, we will allow you to rest in our village for a few days. Upon your departure, however, you are banished from Parthyn. Should you return, we will attack you on sight.”</i>"
```

Comment:
Yup, Malin's arrogant, Drogan is too. Not a great combination when they argue.

Reply:
I think the change makes Drogan sound much weaker and indecisive;

1) There's no need to specify "in these lands", as necromancy is outlawed in essentially all civilized lands he's ever heard of. Why would he be so jovial as to imply that sure you can be a necromancer, just not here?

2) Why would a commander draw attention to his weaker position? Saying "we'd fight you if we were able, but sadly we're at your mercy" makes little sense. Drogan is the person who would rather let the town be sacked than be corrupted by necromancy, so being outnumbered clearly isn't a deal-breaker for him. Drogan has just enough decency that he lets Darken stay and rest (and only once, only this time) as thanks for saving the town, not because he has no other choice.

Overall, I think the original reply by Drogan was better. What's the rationale for the changes to it?

Reply:
I'll implement 1 (just remove the "in these lands", right?). For 2. I intentionally made Drogan's reply weaker and more of a jerk (because that's what I intended to characterize him as). He's not really a very decent or generous man, and certainly not an effective leader.

Reply:
About 1, I might be in the minority here but I don't think "in these lands" is redundant. Drogan, being the captain of the guard of a border town, surely knows that there are necromancers in the lawless north country beyond Wesnoth's borders. He might say "_in these lands_ the penalty for necromancy is death" to mean: you are not in the lawless north country any more, necromancer; you are now in the civilized Kingdom of Wesnoth where you and your kind are killed on sight.